lib/helper/chromeProfileCreator.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+import time
+import shutil
+import psutil
+import tempfile
+import subprocess
+from lib.common.logConfig import get_logger
+from lib.common.pyDriveUtil import PyDriveUtil
+from lib.helper.desktopHelper import close_browser
+
+logger = get_logger(__name__)
+
+
+class ChromeProfileCreator(object):
+    def __init__(self, chrome_profile_path=''):
+        self.browser_type = "chrome"
+        self.process_name = "chrome.exe"
+        self.current_platform_name = sys.platform
+        if self.current_platform_name == 'darwin':
+            self.chrome_cmd = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+        elif self.current_platform_name == "linux2":
+            self.chrome_cmd = 'google-chrome'
+        else:
+            self.chrome_cmd = 'chrome'
+        self._chrome_profile_path = chrome_profile_path
+
+    def get_chrome_profile(self, cookies_settings={}):
+        if self._chrome_profile_path:
+            logger.info('Get Profile: {}'.format(self._chrome_profile_path))
+            return self._chrome_profile_path
+        else:
+            profile_path = self._create_chrome_profile()
+            self._download_cookies(cookies_settings=cookies_settings)
+            return profile_path
+
+    def _create_chrome_profile(self):
+        tmp_profile_dir = tempfile.mkdtemp(prefix='chromeprofile_')
+        logger.info('Creating Profile: {}'.format(tmp_profile_dir))
+        self.chrome_cmd = '{} --user-data-dir={}'.format(self.chrome_cmd, tmp_profile_dir)
```

Comment:
`self.chrome_cmd = '{} --user-data-dir={}'.format(self.chrome_cmd, tmp_profile_dir)`

I am thinking if 
`self.chrome_cmd = '{} --no-startup-window --user-data-dir={}'.format(self.chrome_cmd, tmp_profile_dir)`
could work better.

By doing so, `close_browser(self.browser_type)` can be removed and the chrome process will terminated after created the profiler. I tried in Mac Mini and Windows, and it worked.